Mathlib/CategoryTheory/Limits/FunctorCategory.lean

```diff
@@ -207,60 +213,63 @@ def limitObjIsoLimitCompEvaluation [HasLimitsOfShape J C] (F : J ⥤ K ⥤ C) (k
-  dsimp [limit_obj_iso_limit_comp_evaluation]
-  rw [iso.inv_comp_eq]
+  dsimp [limitObjIsoLimitCompEvaluation]
+  rw [Iso.inv_comp_eq]
   simp
 #align category_theory.limits.limit_obj_iso_limit_comp_evaluation_inv_π_app CategoryTheory.Limits.limitObjIsoLimitCompEvaluation_inv_π_app
 
-@[simp, reassoc.1]
+@[reassoc (attr := simp)]
 theorem limit_map_limitObjIsoLimitCompEvaluation_hom [HasLimitsOfShape J C] {i j : K}
     (F : J ⥤ K ⥤ C) (f : i ⟶ j) :
-    (limit F).map f ≫ (limitObjIsoLimitCompEvaluation _ _).Hom =
-      (limitObjIsoLimitCompEvaluation _ _).Hom ≫ limMap (whiskerLeft _ ((evaluation _ _).map f)) :=
+    (limit F).map f ≫ (limitObjIsoLimitCompEvaluation _ _).hom =
+      (limitObjIsoLimitCompEvaluation _ _).hom ≫ limMap (whiskerLeft _ ((evaluation _ _).map f)) :=
   by
   ext
   dsimp
   simp
 #align category_theory.limits.limit_map_limit_obj_iso_limit_comp_evaluation_hom CategoryTheory.Limits.limit_map_limitObjIsoLimitCompEvaluation_hom
 
-@[simp, reassoc.1]
+@[reassoc (attr := simp)]
 theorem limitObjIsoLimitCompEvaluation_inv_limit_map [HasLimitsOfShape J C] {i j : K}
     (F : J ⥤ K ⥤ C) (f : i ⟶ j) :
     (limitObjIsoLimitCompEvaluation _ _).inv ≫ (limit F).map f =
       limMap (whiskerLeft _ ((evaluation _ _).map f)) ≫ (limitObjIsoLimitCompEvaluation _ _).inv :=
   by
-  rw [iso.inv_comp_eq, ← category.assoc, iso.eq_comp_inv,
-    limit_map_limit_obj_iso_limit_comp_evaluation_hom]
+  rw [Iso.inv_comp_eq, ← Category.assoc, Iso.eq_comp_inv,
+    limit_map_limitObjIsoLimitCompEvaluation_hom]
 #align category_theory.limits.limit_obj_iso_limit_comp_evaluation_inv_limit_map CategoryTheory.Limits.limitObjIsoLimitCompEvaluation_inv_limit_map
 
 @[ext]
 theorem limit_obj_ext {H : J ⥤ K ⥤ C} [HasLimitsOfShape J C] {k : K} {W : C}
     {f g : W ⟶ (limit H).obj k}
     (w : ∀ j, f ≫ (Limits.limit.π H j).app k = g ≫ (Limits.limit.π H j).app k) : f = g := by
-  apply (cancel_mono (limit_obj_iso_limit_comp_evaluation H k).Hom).1
-  ext
+  apply (cancel_mono (limitObjIsoLimitCompEvaluation H k).hom).1
+  ext j
   simpa using w j
 #align category_theory.limits.limit_obj_ext CategoryTheory.Limits.limit_obj_ext
 
 instance evaluationPreservesColimitsOfShape [HasColimitsOfShape J C] (k : K) :
-    PreservesColimitsOfShape J ((evaluation K C).obj k)
-    where PreservesColimit F :=
-    preservesColimitOfPreservesColimitCocone (combinedIsColimit _ _) <|
-      IsColimit.ofIsoColimit (colimit.isColimit _) (evaluateCombinedCocones F _ k).symm
+    PreservesColimitsOfShape J ((evaluation K C).obj k) where 
+  preservesColimit {F} := by
+    let X : (k:K)  → ColimitCocone (Prefunctor.obj (Functor.flip F).toPrefunctor k) := 
+    -- Porting note: now needed X has a hint
+      fun k => getColimitCocone (Prefunctor.obj (Functor.flip F).toPrefunctor k)
+    refine preservesColimitOfPreservesColimitCocone (combinedIsColimit _ _) <|
+      IsColimit.ofIsoColimit (colimit.isColimit _) (evaluateCombinedCocones F X k).symm
```

Comment:
```suggestion
    -- Porting note: added `let` statement, because `X` could not be inferred
    let X : (k:K)  → ColimitCocone (Prefunctor.obj (Functor.flip F).toPrefunctor k) := 
      fun k => getColimitCocone (Prefunctor.obj (Functor.flip F).toPrefunctor k)
    exact preservesColimitOfPreservesColimitCocone (combinedIsColimit _ _) <|
      IsColimit.ofIsoColimit (colimit.isColimit _) (evaluateCombinedCocones F X k).symm
```

Reply:
Changed